src/Xamarin.Android.Build.Tasks/Tasks/ResolveLibraryProjectImports.cs

```diff
@@ -192,6 +208,14 @@ void Extract (
 				string resDir = Path.Combine (importsDir, "res");
 				string resDirArchive = Path.Combine (resDir, "..", "res.zip");
 				string assetsDir = Path.Combine (importsDir, "assets");
+				string nuGetPackageId = assemblyItem.GetMetadata (NuGetPackageId) ?? string.Empty;
+				string nuGetPackageVersion = assemblyItem.GetMetadata (NuGetPackageVersion) ?? string.Empty;
+				Console.WriteLine ($"{assemblyPath}: {nuGetPackageId} {nuGetPackageVersion}");
```

Comment:
Should it use `Log.LogDebugMessage`?

Reply:
Its draft, I'll be cleaning this up later.